compiler/modules/comm.diagnostic/diagnostic.bal

```diff
@@ -112,7 +113,46 @@ public function messageFormat(Template t) returns string {
 
 public function format(Diagnostic d) returns string[] {
     Location loc = d.location;
+    string [] errMessage = [];
```

Comment:
No space here: `string []`

I would call this `lines`.

Reply:
fixed